### ISSUE TITLE
Issue with import/export rule for "multiple export name of" after upgrading to version 2.25.2

### ISSUE THREAD

**Issue by RodrigoOkuta** [import-js/eslint-plugin-import#2285]
I update the `eslint-plugin-import from 2.24.2 to 2.25.2` and I started getting an error "multiple export name of" in the `testing-library/react` file about the custom render. Here is the Testing React setup [guide](https://testing-library.com/docs/react-testing-library/setup/). 

The issue is when you have both export with the `render` name. One comes from `export * from '@testing-library/react'` and the other from the `customRender` which is supposed to override the one from `@testing-library/react`.
![image](https://user-images.githubusercontent.com/25035115/139922727-be78c105-41bf-43b7-8fe9-2f359ca061b7.png)

Is there an issue with the eslint rule `import/export`?

**Comment by ljharb** [comment]
I don’t think it is - the language semantics allow this, but the linter rule can’t know if it’s intentional or not. It seems like the place for an override comment (or, an option that specifically allows this case)
